src/controllers/auth.ts

```diff
@@ -12,16 +12,21 @@ const registerCounter = new client.Counter({
 
 export const registerUser = (req: Request, res: Response) => {
 
-    // we have to send a email that bestätigt that it is ur email
     // send better responses 
```

Comment:
Should we add more informative responses on the front end as well?

Reply:
".. that 'bestätigt' ..." 😂

@thatsddr actually it depends a bit what kind of response @Darjusch means here. Response codes or response messages, second we need to be careful with.

Reply:
@5h3rr1ll a couple of lines after that comment there are messages like: "User does not exist or password/email is wrong"

Reply:
yeah i mean mostly messages but some response codes could also be thought about. 
I think its more for the backend to not give away to much information. 
The example u gave domenico is fine.